RAGchain/preprocess/loader/hwp_table_loader.py

```diff
@@ -0,0 +1,152 @@
+                            'rowspan'] != "1" and flag == 0 and line == 0:
+                            result_txt = result_txt[:-1] + ' rowspan ="{}">'.format(
+                                tag3.attrs['rowspan']) + tag5.get_text()
+                            flag = 1
+                            line = 2
+                        elif tag3.attrs['colspan'] != "1" and tag3.attrs['rowspan'] != "1" and flag == 0:
+                            result_txt = result_txt[:-1] + ' colspan ="{}" rowspan ="{}">'.format(
+                                tag3.attrs['colspan'], tag3.attrs['rowspan']) + tag5.get_text()
+                            flag = 1
+                        else:
+                            result_txt += tag5.get_text()
+                        flag = 0
+                    result_txt += '</td>\n'
+                    line = 0
+        result_txt += '\t </tr>\n'
+    result_txt += '</table>'
+    return result_txt
+
+
+class HwpLoader(BaseLoader):
+    # It only works on Windows
+    def __init__(self, path: str, *args, **kwargs):
+        self.file_path = path
+        self.result = []
+        self.flag = 0
+
+    def convert_hwp_to_hwpx(self):
+        hwp = win32.gencache.EnsureDispatch("HWPFrame.HwpObject")
+        hwp.RegisterModule("FilePathCheckDLL", "FilePathCheckerModule")
+        hwp.HParameterSet.HTableCreation.TableProperties.TreatAsChar = 1
+        hwp.Open(self.file_path)
+        hwp.SaveAs(hwp.Path + "X", "HWPX")
+        hwp.Quit()
+        self.file_path = self.file_path + "x"
+
+    def unzip_hwpx(self, file_path):
+        os.chdir(os.path.dirname(self.file_path))
+        target_path = os.path.join(os.getcwd(), "hwpx")
+        with zipfile.ZipFile(self.file_path, 'r') as zf:
+            zf.extractall(path=target_path)
+        # os.remove(self.file_path)
+
+    def preprocessor(self):
+
+        if self.file_path[-5:] == ".hwpx":
+            self.unzip_hwpx(self.file_path)
+
+        elif self.file_path[-4:] == ".hwp":
+
+            self.convert_hwp_to_hwpx()
+            self.unzip_hwpx(self.file_path)
+
+        else:
+            raise ValueError("The file extension must be .hwp or .hwpx")
+
+        # print("Loading {0}".format(self.file_path))
+
+        pattern = r'</?(?!(?:em|strong)\b)[a-z](?:[^>"\']|"[^"]*"|\'[^\']*\')*>'
+
+        for i, xml in enumerate(spliter(os.path.join(os.getcwd(), "hwpx", "Contents", "section0.xml"))):
+            if i % 2 == 0:
+                self.result.append(re.sub(pattern, '', xml))
+            if i % 2 == 1:
+                self.result.append('<hp:tbl' + xml)
+
+        self.result[0] = self.result[0].strip("""['<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>""")
+
+        for i, xml in enumerate(self.result):
+            if i % 2 == 1:
+                self.result[i] = xml_to_html(xml)
+
+        # print("Preprocessing is done")
+        return self.result
+
+    def load(self) -> List[Document]:
```

Comment:
You must implement lazy_load() first. (You didn't implement this at all)
load() can be just like 
```
return list(self.lazy_load()) 
```

Please watch other loaders that we made.